pkg/pb/orders.proto

```diff
@@ -19,6 +19,7 @@ enum PieceAction {
     GET_REPAIR = 4;
     PUT_REPAIR = 5;
     DELETE = 6;
+    PUT_GRACEFUL_EXIT = 7;
```

Comment:
AFAIR we shouldn't need the `PUT_GRACEFUl_EXIT` action, because the receiver shouldn't distinguish between them.

Reply:
@iglesiasbrandon do we want to track put orders settled by the replacement node differently? or can we just use `PUT`